src/west/commands/common.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2019, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+'''West common commands'''
```

Comment:
Would you mind just putting this in a `help.py` file for now, to make it easy for newcomers to find?

Reply:
Agreed

Reply:
Done